tests/ops/service/privacy_request/test_request_runner_service.py

```diff
@@ -311,6 +312,121 @@ def get_privacy_request_results(
+    args, kwargs = upload_mock.call_args
+    data_category_field_mapping = kwargs["data_category_field_mapping"]
+
+    # make sure the catergory field mapping generally looks as we expect
+    address_mapping = data_category_field_mapping[
+        CollectionAddress.from_string("postgres_example_test_dataset:address")
+    ]
+    assert len(address_mapping) >= 5
+    assert address_mapping["user.contact.address.street"] == [
+        FieldPath("house"),
+        FieldPath("street"),
+    ]
+    product_mapping = data_category_field_mapping[
+        CollectionAddress.from_string("postgres_example_test_dataset:product")
+    ]
+    assert len(product_mapping) >= 1
+    assert product_mapping["system.operations"] == [
+        FieldPath(
+            "id",
+        ),
+        FieldPath(
+            "name",
+        ),
+        FieldPath(
+            "price",
+        ),
+    ]
+
+
+@pytest.mark.integration_postgres
+@pytest.mark.integration
+@mock.patch("fides.api.service.privacy_request.request_runner_service.upload")
+def test_upload_access_results_has_data_use_map(
+    upload_mock: Mock,
+    postgres_example_test_dataset_config_read_access,
+    postgres_integration_db,
+    db,
+    policy,
+    run_privacy_request_task,
+):
+    """
+    Ensure we are passing along a correctly populated data_use_map to the 'upload' function
+    that publishes the access request output.
+    """
+    customer_email = "customer-1@example.com"
+    data = {
+        "requested_at": "2021-08-30T16:09:37.359Z",
+        "policy_key": policy.key,
+        "identity": {"email": customer_email},
+    }
+
+    pr = get_privacy_request_results(
+        db,
+        policy,
+        run_privacy_request_task,
+        data,
+    )
+
+    # sanity check that acccess results returned as expected
+    results = pr.get_results()
+    assert len(results.keys()) == 11
+
+    # what we're really testing - ensure data_use_map arg is well-populated
+    args, kwargs = upload_mock.call_args
+    data_use_map = kwargs["data_use_map"]
+
+    assert data_use_map == {
+        "postgres_example_test_dataset:report": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:employee": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:customer": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:service_request": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:visit": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:address": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:login": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:orders": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:payment_card": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:order_item": "{'marketing.advertising'}",
+        "postgres_example_test_dataset:product": "{'marketing.advertising'}",
+    }
```

Comment:
ah right, highlighting that every node in the dataset will have the same data use.